README.md

```diff
@@ -61,16 +61,19 @@ sudo make install
 with some of them enabled by default (see `BUILDTAGS` in top-level `Makefile`).
 
 To change build tags from the default, set the `BUILDTAGS` variable for make,
-e.g.
+e.g. to disable `seccomp` and enable `nokmem`, run:
 
 ```bash
-make BUILDTAGS='seccomp'
+make BUILDTAGS="nokmem"
```

Comment:
I think we should encourage keeping seccomp enabled; should we change to (e.g.);

```suggestion
e.g. to disable kernel memory limiting, but keep `seccomp`, run:
 
  ```bash
  make BUILDTAGS="nokmem seccomp"
```

Reply:
Doesn't matter much to me, it's just a usage example, not a recommendation about how to compile runc, so can be anything (and I sometimes find myself disabling seccomp so I can compile runc without having to install libseccomp-devel).

But yes, maybe yours is a better/fuller example.

Reply:
Yeah, I agree it's just an example. People love to copy/paste though, and hate to read 😓